.ci/deps.opam.sh

```diff
@@ -1,13 +1,14 @@
 set -e
 set -x
 
-TERM=dumb
+export TERM=dumb
 
 # Infer commands
 if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
   wget -nc -O ~/infer.tar.xz https://github.com/facebook/infer/releases/download/v0.7.0/infer-linux64-v0.7.0.tar.xz
   tar xf ~/infer.tar.xz -C ~/
   cd ~/infer-linux64-v0.7.0
+  apt-get install ocaml ocaml-native-compilers camlp4-extra opam -y
```

Comment:
is this the fix for https://github.com/coala/coala-bears/issues/2006 ?

Reply:
This is not the complete fix,
There were also some other problems relating to the ubuntu-12.04 image in the current circle where we had used:
if [ -n "$ADD_APT_UBUNTU_RELEASE" ]; then
  echo "deb http://archive.ubuntu.com/ubuntu/ $ADD_APT_UBUNTU_RELEASE main universe" | sudo tee -a /etc/apt/sources.list.d/$ADD_APT_UBUNTU_RELEASE.list > /dev/null